server/community/model.ts

```diff
@@ -1,129 +1,131 @@
 
-			navLinks: { type: dataTypes.JSONB },
-			footerLinks: { type: dataTypes.JSONB },
-			footerLogoLink: { type: dataTypes.TEXT },
-			footerTitle: { type: dataTypes.TEXT },
-			footerImage: { type: dataTypes.TEXT },
+		navLinks: { type: dataTypes.JSONB },
+		footerLinks: { type: dataTypes.JSONB },
+		footerLogoLink: { type: dataTypes.TEXT },
+		footerTitle: { type: dataTypes.TEXT },
+		footerImage: { type: dataTypes.TEXT },
 
-			website: { type: dataTypes.TEXT },
-			facebook: { type: dataTypes.TEXT },
-			twitter: { type: dataTypes.TEXT },
-			email: { type: dataTypes.TEXT },
-			issn: { type: dataTypes.TEXT },
-			isFeatured: { type: dataTypes.BOOLEAN },
-			viewHash: { type: dataTypes.STRING },
-			editHash: { type: dataTypes.STRING },
-			premiumLicenseFlag: { type: dataTypes.BOOLEAN, defaultValue: false },
-			defaultPubCollections: { type: dataTypes.JSONB },
-			spamTagId: { type: dataTypes.UUID },
+		website: { type: dataTypes.TEXT },
+		facebook: { type: dataTypes.TEXT },
+		twitter: { type: dataTypes.TEXT },
+		email: { type: dataTypes.TEXT },
+		issn: { type: dataTypes.TEXT },
+		isFeatured: { type: dataTypes.BOOLEAN },
+		viewHash: { type: dataTypes.STRING },
+		editHash: { type: dataTypes.STRING },
+		premiumLicenseFlag: { type: dataTypes.BOOLEAN, defaultValue: false },
+		defaultPubCollections: { type: dataTypes.JSONB },
+		spamTagId: { type: dataTypes.UUID },
 
-			/* Set by Associations */
-			organizationId: { type: dataTypes.UUID },
-		},
-		{
-			classMethods: {
-				associate: (models) => {
-					const {
-						Community,
-						DepositTarget,
-						Organization,
-						Collection,
-						Page,
-						Pub,
-						ScopeSummary,
-						SpamTag,
-					} = models;
-					Community.belongsTo(Organization, {
-						onDelete: 'CASCADE',
-						as: 'organization',
-						foreignKey: 'organizationId',
-					});
-					Community.hasMany(Collection, {
-						onDelete: 'CASCADE',
-						as: 'collections',
-						foreignKey: 'communityId',
-					});
-					Community.hasMany(Pub, {
-						onDelete: 'CASCADE',
-						as: 'pubs',
-						foreignKey: 'communityId',
-					});
-					Community.hasMany(Page, {
-						onDelete: 'CASCADE',
-						as: 'pages',
-						foreignKey: 'communityId',
-					});
-					Community.hasMany(DepositTarget, {
-						onDelete: 'CASCADE',
-						as: 'depositTargets',
-						foreignKey: 'communityId',
-					});
-					Community.belongsTo(ScopeSummary, {
-						as: 'scopeSummary',
-						foreignKey: 'scopeSummaryId',
-					});
-					Community.belongsTo(SpamTag, {
-						as: 'spamTag',
-						foreignKey: 'spamTagId',
-					});
-				},
+		/* Set by Associations */
+		organizationId: { type: dataTypes.UUID },
+	},
+	{
+		// @ts-expect-error ts(2345): Argument of type '{ classMethods: { associate: (models: any) => void; }; }' is not assignable to parameter of type 'ModelOptions<Model<any, any>>'. Object literal may only specify known properties, and 'classMethods' does not exist in type 'ModelOptions<Model<any, any>>'.
```

Comment:
since the sequelize define's now have proper typechecking, this raises an error, so we have to `ts-expect-error` it